src/main/groovy/org/mockito/release/internal/gradle/VersioningPlugin.java

```diff
@@ -62,4 +68,14 @@ public void execute(BumpVersionFileTask t) {
             }
         });
     }
+
+    private void createVersionPropertiesFile(Project project, File versionFile) {
+        LOG.lifecycle("  Required file version.properties doesn't exist. Creating it automatically. Remember about checking it into VCS!");
```

Comment:
Let's also write that the user _should not_ configure the version in the 'build.gradle' file. Only version.properties should contain the version.